### DISCUSSION TITLE
How to handle .yarnrc.yml file when it contains secret info?

### DISCUSSION THREAD

**Discussion by Yohanna** [yarnpkg/berry#2771] [Q&A] [answered]
This a follow up to my earlier question here #2756. If my `.yarnrc.yml` contains auth info for private registries, then I can't commit it. At the same time `.yarnrc.yml` is needed to tell other project members which version of Yarn to use. What's the best way to handle situations like this?

Thank you :)

**Comment by arcanis** [comment, marked as answer]
Store passwords etc in your home folder's `.yarnrc.yml` (and only keep "public" settings in the project-local one).

**Reply by Yohanna** [reply]
Thanks, Maël. I'll try that.
